### ISSUE TITLE
Displaying remaining oxygen, and hiding free breathing (water breathing) effect.

### ISSUE THREAD

**Issue by LoSilopante** [LoSilopante/create-air#1]
Even thought air is drained from the diving tank, there is no display for remaining oxygen time in the UI. 
Having the free breathing effect (water breathing renamed by Thin Air) hidden is another feature that should be enabled when air is drained.

**Comment by dandalous** [comment]
First off, great mod! Much appreciated. Secondly, I noticed Cold Sweat's changelog mentioned that the Create Netherite Backtank can now give you resistance to heat by draining it, while also displaying the remaining time. I checked their code and this seems to be what you're looking for https://github.com/Momo-Softworks/Cold-Sweat/blob/e9beda18d58d50c6becc31cf3f7e0577e151e7c2/src/main/java/com/momosoftworks/coldsweat/util/compat/CompatManager.java#L317

Lastly, would there be any way to add NeoForge (47.1.106) support? As this is the only mod that requires Forge 47.2.0 in my modlist. Thanks again!

**Comment by LoSilopante** [comment]
Thank you! I'll take a look, hoping implementing their way doens't somehow crash the game like the 20 other fixes I've tried...
___
Regarding NeoForge (best modloader ngl), I've made the mod before it was popular but I'd gladly port it over, however I'm currently on vacation and I'd be away from my computer 'til mid September. If you need the mod working before then I think you can try editing line 20 in mods.toml to allow the mod running on lower versions of Forge; I think that should make it run on current NeoForge.

**Comment by dandalous** [comment]
No problem. Enjoy your vacation my dude! From looking at the code comments, seems they don't understand why it works either. Cold Sweat is a decently professional mod which should say a lot that the hack isn't that straightforward

**Comment by RyanBoio** [comment]
Hi there did you ever find a fix to this "glitch" "error" "bug"? (idk what it would be called). 
love your mod and thank you for working on it!

**Comment by LoSilopante** [comment]
Yeah, there's the "possibility" that I've deleted the files for the update that I've been working on when clealing up my pc, I don't think I'll get back into making mods anytime soon.
It's not as much as a bug as it is a feature that I hadn't implemented yet.

**Comment by dandalous** [comment]
There are data recovery tools for retrieving deleted files. In the meantime you can use Cold Sweat (but hide its menu and turn its difficulty off if you want) as it already (indirectly) implements this feature for Thin Air/Create Air

**Comment by LoSilopante** [comment]
Fixed
